Dockerfile

```diff
@@ -49,12 +49,6 @@ RUN mkdir -p ~/scripts
 COPY scripts /scripts
 RUN echo "source /scripts/dotnetcore.sh" >> ~/.bashrc
 
-# install reportgenerator for code coverage
-RUN dotnet tool install -g dotnet-reportgenerator-globaltool
```

Comment:
Why did you move this to the `dotnet-build.sh` file?

This way it will execute the tool install **every time** you run the container.
Tools should really be included in the container so we have a static and replayable environment between builds

Reply:
I removed them because they sometimes resulted in errors because they could not not be updated because it was a fixed container layer.

Normally for dotnet tools this is done using a [local tool manifest file](https://learn.microsoft.com/en-us/dotnet/core/tools/local-tools-how-to-use#restore-a-local-tool-installed-by-others) in the repo just like package.json. They are also not in the container.

Plus the "tool install every time" can be partly solved with using the bitbucket pipeline cache better.